ember-app/app/templates/issue/reference.hbs

```diff
@@ -11,6 +11,9 @@
      <span class="commit-message"> 
        <span class="commit-link">-o-</span> {{hb-avatar user=model.actor width=16 height=16 }} 
        {{view.message}}
+       {{#unless view.message}}
+         <span class="empty">No commit message</span>
+       {{/unless}}
```

Comment:
Any ideas why this block isn't being hit? I added logging to confirm that `message` is, in fact, empty.